src/Content/OEmbed.php

```diff
@@ -326,7 +328,7 @@ public static function getHTML($url, $title = null)
 
 		$o = self::fetchURL($url, !self::isAllowedURL($url));
 
-		if (!is_object($o) || $o->type == 'error') {
+		if (!is_object($o) || !property_exists($o, 'type') || $o->type == 'error') {
```

Comment:
This isn't the right condition. When we check for a specific value, we want the property to be existing and it to equals the value. It should be rather:
```php
if (!is_object($o) || property_exists($o, 'type') && $o->type == 'error') {
```

Reply:
Oh, dear. 02:51 a.m. edits ... ;-)

Reply:
My comment was written at 1:00 AM, I'm relating.

Reply:
Fixed the condition check.